runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -1312,4 +1314,12 @@ public void onScrollChangedDelegate(int l, int t, int oldl, int oldt) {
               preWrapperLines = {"onFocusChanged(gainFocus, direction, previouslyFocusedRect);"})
     public void onFocusChangedDelegate(boolean gainFocus, int direction, Rect previouslyFocusedRect) {
     }
+
+    // Override XWalkView.setOnTouchListener to install the listener to ContentView
+    // therefore touch event intercept through onTouchListener is available on XWalkView.
+    @Override
+    @XWalkAPI
+    public void setOnTouchListener(OnTouchListener l) {
```

Comment:
Actually it does appear in java doc as long as you add `@XWalkAPI` onto it. How ever, if you remove this mark, the reflection layer won't support this interface. I agree that it's not necessary add `@since` is it's not an embedding API. You can add comment like `See android.view.View.OnTouchListener` instead of leaving the description blank.